tests/test_fields.py

```diff
@@ -24,6 +24,13 @@ class Aux(models.Model):
     field_foreign = models.ForeignKey('Aux', null=True, on_delete=models.CASCADE)
 
 
+class AuxSerializer(serializers.ModelSerializer):
```

Comment:
I was unable to move the serializer back to the original location since it's still used in the `AllFieldsSerializer` class. Let me know if I missed something!

Reply:
sry my bad. you are right